cmd/ssh3/main.go

```diff
@@ -47,21 +47,24 @@ func homedir() string {
 // If non-nil, use udpConn as transport (can be used for proxy jump)
```

Comment:
This parameter no longer exists
```suggestion
// If non-nil, establishes a proxy connection towards proxyRemoteAddr first
```